### PR TITLE
Update versions for Spring obo release

### PR DESCRIPTION
**Pull request by backwind1233**

### CODE REVIEW

**Review by stliu — APPROVED** [review]
LGTM

File: sdk/spring/azure-spring-boot-test-aad/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-aad/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-application/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-application/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-core/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-core/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-cosmos/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-cosmos/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-keyvault/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-keyvault/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-parent/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-parent/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-selenium-common/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-selenium-common/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-servicebus-jms/CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-servicebus-jms/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-boot-test-storage/CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-test-storage/CHANGELOG.md:3]: Here.

File: sdk/spring/azure-spring-cloud-test-eventhubs/CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,3 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2021-03-03)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-test-eventhubs/CHANGELOG.md:3]: It will not be released.

**Review by chenrujun — CHANGES_REQUESTED** [review]

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/spring/azure-spring-cloud-test-eventhubs/CHANGELOG.md (hunk 10 above)
Comment (eightstars) [reply sdk/spring/azure-spring-cloud-test-eventhubs/CHANGELOG.md]: よくできました

File: sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.0.0-beta.1 (Unreleased)
+## 3.0.0-beta.1 (2021-03-03)
```
[the remark this answers is not in the document]
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md]: Hi, @backwind1233 , it should not be released.

**Pull request merged**